components/FormStepper.tsx

```diff
@@ -1,47 +1,69 @@
 import * as React from "react";
-import { ContentBlockWide } from "../styles";
-import styled from "styled-components";
+import { Container, Panel } from "../styles";
+import styled, { DefaultTheme } from "styled-components";
 
 type Props = {
   serverStep: number;
   steps: FormStep[];
+  user?: Object;
```

Comment:
Is the user object here expected to be a hackerProfile?

Reply:
actually, yeah, was just going off what we had atm -- will refactor this in a follow-up